Mono.Cecil/ModuleDefinition.cs

```diff
@@ -240,6 +240,8 @@ public sealed class WriterParameters {
 
 	public sealed class ModuleDefinition : ModuleReference, ICustomAttributeProvider, ICustomDebugInformationProvider, IDisposable {
 
+		private static ushort defaultLinkerValue = 0; // or 8 to retain previous default behaviour
```

Comment:
I think we can remove this safely in favor of using `8` in the declaration, see below.